exporters/stats/prometheus/src/main/java/io/opencensus/exporter/stats/prometheus/PrometheusExportUtils.java

```diff
@@ -184,8 +183,8 @@ public Void apply(CountData arg) {
           @Override
           public Void apply(DistributionData arg) {
             // For histogram buckets, manually add the bucket boundaries as "le" labels.
-            // See io.prometheus.client.Histogram.java#L329.
-            checkLeLabelInLabelNames(labelNames);
+            // See github.com/prometheus/client_java/blob/master/
+            // simpleclient/src/main/java/io/prometheus/client/Histogram.java#L329
```

Comment:
The two Github links in comments should also specify commits, in case the code changes.